Parser/StartupConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Parser.Infrastructure;
+using Parser.Services;
+
+namespace Parser
+{
+    public static class StartupConfiguration
+    {
+        public static void InjectDependencies(this IServiceCollection services)
+        {
```

Comment:
The good practice is to have dependency injection module for every aggregate. Since your project is really tiny so far this class looks excessive.